packages/sample-app/src/components/common/PageHeader.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  Brand,
+  Button,
+  Masthead,
+  MastheadToggle,
+  MastheadMain,
+  MastheadBrand,
+  MastheadContent,
+  PageToggleButton,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+  ToolbarGroup,
+} from '@patternfly/react-core';
+import { BarsIcon } from '@patternfly/react-icons';
+import * as React from 'react';
+import pfLogo from '../../images/pfColorLogo.svg';
+import LoadPluginModal from './LoadPluginModal';
+import type { LoadPluginModalRefProps } from './LoadPluginModal';
+
+const PageHeader: React.FC = () => {
+  const loadPluginModalRef = React.useRef<LoadPluginModalRefProps>(null);
+
+  const openLoadPluginModal = () => {
+    loadPluginModalRef?.current?.open();
+  };
+
+  return (
+    <Masthead>
+      <MastheadToggle>
+        <PageToggleButton variant="plain">
+          <BarsIcon />
+        </PageToggleButton>
+      </MastheadToggle>
+      <MastheadMain>
+        <MastheadBrand>
+          <Brand src={pfLogo} alt="PatternFly logo" />
+        </MastheadBrand>
+      </MastheadMain>
+      <MastheadContent>
+        <Toolbar>
```

Comment:
FWIW this is a large aggregation of components. We may want to consider moving some to their own files respectively (for customization), but for now this is fine.